billy_metadata/sd.py

```diff
@@ -0,0 +1,98 @@
+import datetime
+
+
+settings = dict(
```

Comment:
Do you know if we still need these settings? If so, they should be in `openstates/sd/__init__.py`.